.gitlab-ci.yml

```diff
@@ -77,25 +79,30 @@ check:runtime:
 
 test:rust:stable:                  &test
   stage:                           test
+  cache:
+    key:                           "${CI_JOB_NAME}-test"
+    paths:
+      - ${CARGO_HOME}
+      - ./target
   variables:
     RUST_TOOLCHAIN: stable
     # Enable debug assertions since we are running optimized builds for testing
     # but still want to have debug assertions.
     RUSTFLAGS: -Cdebug-assertions=y
     TARGET: native
+  # <<:                              *kubernetes_build
+  tags:
+    - linux-docker
   only:
     - tags
     - master
     - schedules
     - web
-    - /^pr-[0-9]+$/
```

Comment:
not on PRs anymore?

Reply:
synching between github and gitlab has been changed and pull requests cannot be prepended with `pr-` string anymore. That's why there is the new one below.